### PR TITLE
frost-core: expose SigningKey::to/from_scalar()

### LINKED ISSUE #497: `SigningKey::{from_scalar, to_scalar}` methods are inconveniently hidden
These methods are gated by an `internals` feature:
```rust
    /// Creates a SigningKey from a scalar.
    #[cfg(feature = "internals")]
    pub fn from_scalar(
        scalar: <<<C as Ciphersuite>::Group as Group>::Field as Field>::Scalar,
    ) -> Self {
        Self { scalar }
    }

    /// Return the underlying scalar.
    #[cfg(feature = "internals")]
    pub fn to_scalar(self) -> <<<C as Ciphersuite>::Group as Group>::Field as Field>::Scalar {
        self.scalar
    }
```
We want to use these methods to bridge between our existing key types and the FROST types; it'd be convenient if the methods were exposed by default.

I don't think there's any meaningful increase in encapsulation, since the secret is already exposed via the serialization methods.

### PR DESCRIPTION
**Pull request by conradoplg**
Closes #497

### CODE REVIEW

File: frost-core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Entries are listed in reverse chronological order.
 
 ## Unreleased
 
+* Exposed the `SignignKey::from_scalar()` and `to_scalar()` methods. This
```
Comment (natalieesk) [inline frost-core/CHANGELOG.md:7]: typo `SigningKey::from_scalar()`

**Review by natalieesk — CHANGES_REQUESTED** [review]
Just a typo

File: frost-core/CHANGELOG.md (hunk 1 above)
Comment (conradoplg) [inline frost-core/CHANGELOG.md:7]: Oops, thanks!

```suggestion
* Exposed the `SigningKey::from_scalar()` and `to_scalar()` methods. This
```

Comment (conradoplg) [reply frost-core/CHANGELOG.md]: Done in 6ba38c3

**Review by natalieesk — APPROVED** [review]
Great

**Pull request merged**